examples/fullscreen.rs

```diff
@@ -1,56 +1,35 @@
-use std::io::{self, Write};
-use winit::{
-    event::{ElementState, Event, KeyboardInput, VirtualKeyCode, WindowEvent},
-    event_loop::{ControlFlow, EventLoop},
-    monitor::MonitorHandle,
-    window::WindowBuilder,
-};
+use std::io::{stdin, stdout, Write};
+use winit::event::{ElementState, Event, KeyboardInput, VirtualKeyCode, WindowEvent};
+use winit::event_loop::{ControlFlow, EventLoop};
+use winit::monitor::{MonitorHandle, VideoMode};
+use winit::window::{Fullscreen, WindowBuilder};
 
 fn main() {
     let event_loop = EventLoop::new();
 
-    #[cfg(target_os = "macos")]
-    let mut macos_use_simple_fullscreen = false;
-
-    let monitor = {
-        // On macOS there are two fullscreen modes "native" and "simple"
-        #[cfg(target_os = "macos")]
-        {
-            print!("Please choose the fullscreen mode: (1) native, (2) simple: ");
-            io::stdout().flush().unwrap();
-
-            let mut num = String::new();
-            io::stdin().read_line(&mut num).unwrap();
-            let num = num.trim().parse().ok().expect("Please enter a number");
-            match num {
-                2 => macos_use_simple_fullscreen = true,
-                _ => {}
-            }
-
-            // Prompt for monitor when using native fullscreen
-            if !macos_use_simple_fullscreen {
-                Some(prompt_for_monitor(&event_loop))
-            } else {
-                None
-            }
-        }
+    print!("Please choose the fullscreen mode: (1) exclusive, (2) borderless: ");
+    stdout().flush().unwrap();
 
-        #[cfg(not(target_os = "macos"))]
-        Some(prompt_for_monitor(&event_loop))
-    };
+    let mut num = String::new();
+    stdin().read_line(&mut num).unwrap();
+    let num = num.trim().parse().ok().expect("Please enter a number");
+
+    let fullscreen = Some(match num {
```

Comment:
`fullscreen` is an option because an `Option<Fullscreen>` is what we have to pass into `with_fullscreen` and `set_fullscreen`. It doesn't really matter if we make it into `Some` here or at the call sites.

Reply:
Yikes, that's a silly complaint from me. [I misread this code](https://github.com/rust-windowing/winit/pull/925/files/b6eb841ff094efa3a85a9dfea4d4b114b9b05bcd#r298030273), it should be fine actually.